src/main/java/com/monitoring/scheduler/DeploymentAndHPAScheduler.java

```diff
@@ -0,0 +1,100 @@
+package com.monitoring.scheduler;
+
+import java.io.IOException;
+import java.security.KeyManagementException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+
+import org.codehaus.jackson.map.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.monitoring.config.DeploymentConfig;
+import com.monitoring.config.EnvironmentComponent;
+import com.monitoring.config.HPAConfig;
+import com.monitoring.dto.Environment;
+import com.monitoring.dto.Deployments.Deployment;
+import com.monitoring.dto.HPA.HorizontalPodAutoscaler;
+import com.monitoring.dto.NameSpaces.Item;
+import com.monitoring.dto.NameSpaces.NameSpaces;
+import com.monitoring.utility.ProjectUtility;
+
+@Component
+public class DeploymentAndHPAScheduler {
+	
+	@Autowired
+	EnvironmentComponent environmentConifg;
+	@Autowired
+	DeploymentConfig deploymentConfig;
+	@Autowired
+	HPAConfig hpaConfig;
+	@Value("#{'${skippedNameSpaces}'.split(',')}")
+	private List<String> skippedNameSpaces;
+	
+	ObjectMapper objectMapper;
+	
+	private static final Logger log = LoggerFactory.getLogger(DeploymentAndHPAScheduler.class);
+	
+	@Scheduled(fixedDelay = 3600000, initialDelay = 5000)
```

Comment:
Keep schedule as a config rather putting in fixedDelay

Reply:
done